gp-includes/things/glossary.php

```diff
@@ -182,11 +182,12 @@ public function delete() {
 	 * @return GP_Project The project
 	 */
 	public function get_locale_glossary_project() {
+		$locale_glossary_prefix = apply_filters( 'gp_locale_glossary_path_prefix', '/languages' );
 		return new GP::$project( array(
 			'id'   => 0,
 			'name' => 'Locale Glossary',
 			'slug' => 0,
-			'path' => '//languages',
+			'path' => "/$locale_glossary_prefix",
```

Comment:
That's a hack for https://github.com/GlotPress/GlotPress-WP/blob/2.3.0/gp-includes/url.php#L149 :)